db/migrate/20220118221746_create_feeds.rb

```diff
@@ -0,0 +1,43 @@
+class CreateFeeds < ActiveRecord::Migration
+  def change
+    remove_column :podcasts, :max_episodes, :integer
+
+    create_table :feeds do |t|
+      t.references :podcast, index: true, foreign_key: true
+      t.string :label
+      t.string :slug, null: false
+
+      # optional overrides
+      t.text :title
+      t.string :url
+      t.string :path
+      t.string :enclosure_prefix
+      t.string :enclosure_template
+      t.integer :display_episodes_count
+      t.integer :display_full_episodes_count
+
+      # optional sub-feed settings
+      t.boolean :private, default: true
+      t.integer :publish_offset_seconds
+      t.text :reject_zones
+      t.text :filter_tags
```

Comment:
Another JSON encoded list of tags, but these are the episodes that get _included_ in the RSS.  So these are interpreted 100% by Feeder itself.

I called them `tags` thinking we'd want to check both Episode categories and keywords for these.  But I could be convinced otherwise.